utils/model_uploader/trigger_ml_models_release.sh

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+# This program is run by "Model Auto-tracing & Uploading" 
+# & "Model Listing Uploading" workflow (See model_uploader.yml 
+# & model_listing_uploader.yml) to trigger ml-models-release 
+# Jenkins workflow.
+
+JENKINS_TRIGGER_TOKEN=$1
+JENKINS_PARAMS=$2
+JENKINS_URL="https://build.ci.opensearch.org"
+
+TIMEPASS=0
+TIMEOUT=7200
```

Comment:
Signing and copying should not take more than few minutes. Recommending to reduce the timeout to 1 hour
```suggestion
TIMEOUT=3600
```